### PR TITLE
Updated country field to be a dropdown on manual address entry screen

### PR DESCRIPTION
**Pull request by ttingle-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by cbroadley-ch — DISMISSED** [review]
LGTM

File: locales/cy/common-address-manual.json [hunk 1]
```diff
@@ -11,21 +11,25 @@
     "addressPostcode": "Cod post",
     "infoShowOnPublic": "Pa wybodaeth byddwn yn dangos ar y rhestr gyhoeddus o asiantau awdurdodedig",
     "correspondenceLocationInfo": "Byddwn yn dangos tref neu ddinas y cyfeiriad gohebiaeth.",
+    "england": "England",
+    "scotland": "Scotland",
+    "northernIreland": "Northern Ireland",
+    "wales": "Wales",
+    "selectCountry": "Select a country Welsh",
+    
     "error-noPropertyDetails": "Cofnodwch enw neu rif eiddo",
```
[the remark this answers is not in the document]
Comment (ttingle-ch) [reply locales/cy/common-address-manual.json:20]: Updated

**Review by cbroadley-ch — APPROVED** [review]

**Review by sardhani-ch — DISMISSED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by cbroadley-ch — APPROVED** [review]

**Review by sardhani-ch — APPROVED** [review]

**Pull request merged**